bin/move-and-rebuild-host-hil.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
```

Comment:
Is this a copy of an existing script but with the "move hosts" part invoked as a python script?  Could you add a comment to that effect, if so, otherwise, let's try to explain in a comment what the purpose of this script is.